contracthashtool.c

```diff
@@ -170,7 +170,7 @@ int main(int argc, char* argv[]) {
 	if (mode == 0x2 && !priv_key_str)
 		USAGEEXIT("No private key specified\n");
 
-	secp256k1_context_t *secp256k1_ctx = secp256k1_context_create((unsigned int) -1);
+	secp256k1_context_t *secp256k1_ctx = secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
```

Comment:
The pubkey_tweak functions need VERIFY. I'll update the documentation for that indeed.